release.sh

```diff
@@ -655,6 +655,25 @@ function publish_artifacts() {
   banner "New release published successfully"
 }
 
+# Sets the github release with the highest semver to 'latest'
+function set_latest_to_highest_semver() {
+  # List latest release
+  local releases # don't combine with the line below, or $? will be 0
+  # Support tags in two formats
+  # - knative-v1.0.0
+  # - v1.0.0
+  releases="$(hub_tool -p release | cut -d '-' -f2)"
+  if ! [[ $? -eq 0 ]]; then
+    abort "cannot list releases"
+  fi
+  local last_version="$(echo "${releases}" | grep '^v[0-9]\+\.[0-9]\+\.[0-9]\+$' | sort -r -V | head -1)"
```

Comment:
```bash
local xxx="$(false)"
```
is always true. https://www.shellcheck.net/wiki/SC2155

Reply:
which is fine for this line. that is why other variables don't have it all one one line and have comments

Reply:
How is this fine? If any command within that expression, we don't get fail but an empty value as last_version. We expect if any command fails, the script fails.